src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -199,9 +201,7 @@ class UpdateConfig(_AnsiblePlaybookAlias):
     """
 
     def run(self, args, unknown_args):
-        args.playbook = 'deploy_localsettings.yml'
-        unknown_args += ('--tags=localsettings',)
-        return AnsiblePlaybook(self.parser).run(args, unknown_args)
+        return commcare_cloud(args.environment, 'ansible-playbook', 'deploy_localsettings.yml', tags='localsettings', *unknown_args)
```

Comment:
Can you articulate why you think this is better than the way we're doing it now?

Reply:
I think it's clearer and cleaner, and sticks to the public interface of a command. I've often found myself annoyed that if I want a command to be an alias for another bash command, then easy, I just call it with `subprocess.call`; but if I want a command to be an alias for another `commcare-cloud` command, then I have to go through this dance of reconstructing what a call to that command class's `run` method, which requires passing in a parser that doesn't really get used (so we arbitrarily use the one we have on hand), and `Namespace` object as `args`, so we modify the one we have on hand and use it.

This shortcut lets us call the command as we would from the command line, but with just a touch of syntactic sugar (but not enough to cause a syntactic blood sugar spike).

I found myself wanting this again for a new command, and feeling averse to doing the little dance again, so I figured I'd write it.

That's how I came to it. Does any of that resonate?